packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -12,9 +12,22 @@ import { UP, DOWN, ENTER } from '@wordpress/keycodes';
 /**
  * Internal dependencies
  */
-import { LinkControl } from '../index';
+import LinkControl from '../';
 import { fauxEntitySuggestions, fetchFauxEntitySuggestions } from './fixtures';
 
+const mockFetchSearchSuggestions = jest.fn();
+
+jest.mock( '@wordpress/data', () => {
```

Comment:
👍

Reply:
I'm pretty sure we do `requireActual( "@wordpress/*" )` for other packages. I wonder if it's broken there as well or if it works only for some packages.

Reply:
> I'm pretty sure we do `requireActual( "@wordpress/*" )` for other packages. I wonder if it's broken there as well or if it works only for some packages.

Yeah, I didn't actually look in to it too much, but it _was_ resolving to _something_, it just seemed it have been not the intended import file (from npm vs. from working directory? src vs. build vs. build-module?). I/we should probably do more to understand what's actually happening.